src/spootnik/reporter/sentry.clj

```diff
@@ -59,28 +60,30 @@
   Supported event keys:
   https://github.com/getsentry/sentry-clj/tree/master?tab=readme-ov-file#supported-event-keys
   "
-
   [e options tags]
-  (let [{:keys [message extra throwable]} e
+  (let [{:keys [message extra throwable fingerprint]} e
         message      (or message (ex-message e))
         user         (some-> extra :org/uuid str)
-        fingerprints (some-> options :fingerpint seq)
-        request      (some-> extra :payload payload->sentry-request)]
+        fingerprints (some-> fingerprint seq)
+        request      (some-> extra :payload payload->sentry-request)
+        ;; override "base" tags with params
+        merged-tags  (merge (:tags options) tags)]
 
     (cond-> {:message message
              :level :error
              :platform "java"
              :server-name  (localhost)}
 
-      throwable    (assoc :throwable throwable)
-      extra        (assoc :extra extra)
-      (seq tags)   (assoc :tags tags)
-      user         (assoc :user user)
-      fingerprints (assoc :fingerprints fingerprints)
-      request      (assoc :request request))))
+      throwable         (assoc :throwable throwable)
+      extra             (assoc :extra extra)
+      (seq merged-tags) (assoc :tags merged-tags)
+      user              (assoc :user user)
+      fingerprints      (assoc :fingerprints fingerprints)
+      request           (assoc :request request))))
 
-(defn send-event! [dsn options e tags]
-  (let [event (e->sentry-event e options tags)]
+(defn send-event! [{:keys [dsn] :as sentry} legacy-options e tags]
+  ;; sentry-options are legacy
```

Comment:
Those are called `legacy-options`, I would remove the comment.

Reply:
done